meson.build

```diff
@@ -0,0 +1,102 @@
+project('rauc-hawkbit-updater', 'c',
+	 version : '1.0',
```

Comment:
You can add `meson_version: '>=0.50'` here and meson will use an informative error message if people try to build this with too old versions of meson.

Meson will also warn you when you use meson functionality that is newer than 0.50 without bumping the minimum version.

Reply:
Added as suggested.